### ISSUE TITLE
Tweak stale and lock bot timeframes to reduce impact

### ISSUE THREAD

**Issue by jezdez** [conda/infra#547]
It became clear that we may want to expedite the locking and stale-marking for the conda-build repo (at least) to reduce the impact on community members that have participated in a lot of issues in the past.

@isuruf If you have more details of the anti-patterns of notification (we can't disable them completely AFAIK), please let me know.

**Comment by jezdez** [comment]
Agreed, thanks @isuruf!

**Comment by kenodegard** [comment]
In addition to our [self-imposed 100 issues/PRs stale limit per run](https://github.com/conda/infra/blob/41e244ca8415eab66880da0c8a1f20b12406e04c/.github/workflows/stale.yml#L67), the lock action appears to have a [hardcoded 50 locks limit per run](https://github.com/dessant/lock-threads#why-are-only-some-issues-and-pull-requests-processed) to avoid GitHub's rate limits.

@isuruf @jezdez this makes me inclined to not spend time trying to speed things up **but** we can instead remove the close comment and/or lock comment? Issues/PRs will continue to be marked as stale and eventually locked at the current rate but you'll get 1-2 fewer emails per thread.

To summarize, our stale/close/lock workflows currently produce 4 notifications/emails:
1. stale comment
2. closing comment
3. closing action
4. locking comment

We can eliminate 2 & 4.

**Comment by isuruf** [comment]
> I don't feel inclined to spend time trying to speed things up but we can instead remove the close comment and/or lock comment?

Why? Do you think it's reasonable to annoy your oldest contributors?

> We can eliminate 2 & 4.

Does step 3 not produce a notification or email? If so, that's good enough

**Comment by kenodegard** [comment]
> > I don't feel inclined to spend time trying to speed things up but we can instead remove the close comment and/or lock comment?
> 
> Why? Do you think it's reasonable to annoy your oldest contributors?

@isuruf My apologies, this was not my intention to convey. We do not wish to annoy any contributor, old or new.

I sought to highlight that seeking to speed this up would require custom implementations that would need to be maintained, a choice we'd wish to avoid if the alternative option proposed was sufficient. As noted we are also limited by GitHub's rate-limiting rules, so these soft/hardcoded limits aren't necessarily all that adjustable in the first place especially when we are performing these maintenance workflows across almost all of the conda org repos.

> > We can eliminate 2 & 4.
> 
> Does step 3 not produce a notification or email? If so, that's good enough

The act of closing an issue/PR produces a GitHub notification, we have no control over this.

So the proposal is to change the workflows from producing 4 emails to only producing 2, one when @conda-bot announces the issue/PR is stale and a second one when @conda-bot closes the issue/PR sometime later. Since the act of locking an issue does not produce an email, you will only see these 2 stale/closing emails on issues/PRs that are still open today. Any issues/PRs you have participated in that have been already closed and are then locked at some point in the future will not produce any emails.
